inst/traduire/pt-translation.json

```diff
@@ -309,7 +309,7 @@
     "TYPE_SPECTRUM": "spectrum",
     "TYPE_UNADJUSTED": "Não ajustado",
     "WARNING_OPTIONS_MISSING_ARTATTEND": "Optou por se adaptar ao modelo sem estimar a assistência TARV vizinha. Poderá querer rever a sua selecção para incluir esta opção.",
-    "WARNING_OUTPUTS_PREV_EXCEEDS_THRESHOLD": "A prevalência do VIH é superior a 40% para algumas dissagregações de idade/sexo",
-    "WARNING_OUTPUTS_ARTCOV_EXCEEDS_THRESHOLD": "A cobertura TARV é superior a 100% para algumas dissagregações de idade/sexo",
+    "WARNING_OUTPUTS_PREV_EXCEEDS_THRESHOLD": "A prevalência do VIH é superior a 40% para: ",
```

Comment:
This is giving a double space `"  "`, one from here and one from the `paste0` `msg <- paste0(t, " ", strat)`